docs/interfaces/dmx-output.md

```diff
@@ -31,21 +31,21 @@ For the DMX feature to work, you'll need to compile WLED from source. It's not a
    `#define WLED_ENABLE_DMX`
    Yes, you just remove the //, which enables the line and therefore DMX support.
 
-3. change either the DMX output pin (sendPin in src/dependencies/ESPDMX.cpp) or the LED output pin (LEDPIN in NpbWrapper.h) to something other than 2. If both are set to the same setting, you might experience slight flickering on your DMX output.
+3. By default the DMX output tries to use pin 2 this conflicts with the LEDPIN, so either change either the DMX output pin (sendPin in src/dependencies/ESPDMX.cpp) or the LED output pin (LEDPIN in NpbWrapper.h) to something other than 2. If both are set to the same setting, you might experience slight flickering on your DMX output.
```

Comment:
Incorrect. LEDPIN is defined in `const.h` and changes to 1 if DMX is compiled.

```c++
//this is merely a default now and can be changed at runtime
#ifndef LEDPIN
#if defined(ESP8266) || (defined(ARDUINO_ARCH_ESP32) && defined(WLED_USE_PSRAM)) || defined(CONFIG_IDF_TARGET_ESP32C3)
  #define LEDPIN 2    // GPIO2 (D4) on Wemod D1 mini compatible boards
#else
  #define LEDPIN 16   // aligns with GPIO2 (D4) on Wemos D1 mini32 compatible boards
#endif
#endif

#ifdef WLED_ENABLE_DMX
#if (LEDPIN == 2)
  #undef LEDPIN
  #define LEDPIN 1
  #warning "Pin conflict compiling with DMX and LEDs on pin 2. The default LED pin has been changed to pin 1."
#endif
#endif
```

Reply:
So are the pre-existing comments in the doc entirely redundant then and should be removed instead?

Reply:
Perhaps not, some may be just outdated or obsolete and some are still valid.

Reply:
So just how to change the pin, but remove reference to the clash with the led?

Reply:
There is no more clash as precompiler directives take care of that.
The other thing is, there is no more NbpWrapper.h.

BTW I did not write the docs, I am merely describing what is in the code. So if you are to update the docs please use what I provide.